Sloth.Api/Startup.cs

```diff
@@ -99,46 +98,56 @@ public void ConfigureServices(IServiceCollection services)
             // add swagger/swashbuckler
             services.AddSwaggerGen(c =>
             {
-                c.SwaggerDoc("v1", new Info
+                c.SwaggerDoc("v1", new OpenApiInfo
                 {
                     Title = "Sloth API v1",
                     Version = "v1",
                     Description = "Scrubber Loader & Online Transaction Hub",
-                    Contact = new Contact()
+                    Contact = new OpenApiContact
                     {
                         Name = "John Knoll",
                         Email = "jpknoll@ucdavis.edu"
```

Comment:
@sprucely While you are here, you might as well change this.

Reply:
@jSylvestre Change it to what? Scott?
@srkirkland I see these hard-coded contacts everywhere. When I see them, should I just automatically convert them to config options? It would be a bit of extra work, but the pattern is straight forward.

Reply:
@sprucely config is a good idea but if these are just like contact for help then I think we should just put in the `apprequests@caes.ucdavis.edu` address and call it a day.